app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    command_list = command.split()
```

Comment:
you can unpack these variables in the easiest way, also use the more descriptive names